src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  return function(a) {
+    callCounter++;
+
+    if (callCounter <= 3) {
+      return function(b) {
+        return a + b;
+      };
+    } else if (callCounter % 2 === 0) {
+      return function() {
+        return 'Bzzz... Error!';
+      };
+    } else {
+      return function(b) {
+        return a + b;
```

Comment:
You wrote it too complicated, you don't need to use if else here, you can combine conditions
This is a closure, you must have a variable in the outer function that stores your result after the inner function is executed